teamup-server/src/main/java/com/project/teamup/controller/UserController.java

```diff
@@ -42,4 +42,11 @@ public UserDTO findById(@PathVariable("id") Long id) {
     public void deleteUser(@PathVariable("id") Long id) {
         userService.delete(id);
     }
+
+    @PostMapping(value = "/activate")
+    public String activateUser(@RequestBody List<UserDTO> users){
+        User deactivatedUser = userMapper.toEntity(users.get(0));
```

Comment:
Please use as admin the user that requested activateUser

Reply:
Please explain

Reply:
You could get the authenticated user from the security context or you could update the parameter list to receive the logged user id

Reply:
And also update the params, it would be better to have as params userId and adminId instead of UserDTO list. If you need help solving the comments let me know :)

Reply:
I think it is easier to have as params the usernames of the both users, instead of having the id's. Can I put usernames instead?

Reply:
My point was that you don`t really need the UserDTO for this action so yes, you could use the usernames